utils/windows-smoke-tests/smoke_test.ps1

```diff
@@ -0,0 +1,30 @@
+# Attempts to build and test `swift-win32`.
+$ErrorActionPreference = 'Stop'
+
+Write-Host "Starting smoke test"
+Write-Host ""
+
+swift --version
+Write-Host ""
+
+$RepoUrl = "https://github.com/compnerd/swift-win32"
+$CloneDir = "C:\swift-win32"
+
+Write-Host "Cloning $RepoUrl"
+git clone $RepoUrl $CloneDir
+if ($LASTEXITCODE -ne 0) { exit $LASTEXITCODE }
+Write-Host ""
+
+Set-Location $CloneDir
+
+Write-Host "Building swift-win32"
+swift build
```

Comment:
suggestion: can we inspect the output and ensure it's not empty?  I'm asking is this would not have caught the issue reported by:

- https://github.com/swiftlang/swift/issues/85252
- https://github.com/swiftlang/swift/issues/80356